typings install postinstall is broken

```
npm ERR! Darwin 15.3.0
npm ERR! argv "/node/v5.5.0/bin/node" "/node/v5.5.0/bin/npm" "install" "--save" "angularfire2"
npm ERR! node v5.5.0
npm ERR! npm  v3.6.0
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn

npm ERR! angularfire2@2.0.0-alpha.1 postinstall: `typings install`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the angularfire2@2.0.0-alpha.1 postinstall script 'typings install'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the angularfire2 package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     typings install
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs angularfire2
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls angularfire2
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Projects/angularfire-demo-2851/npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -1,14 +1,17 @@
 {
   "name": "angularfire2",
-  "version": "2.0.0-alpha.1",
+  "version": "2.0.0-alpha.2",
   "description": "",
   "main": "dist/angularfire.js",
   "scripts": {
     "test": "npm run build; karma start",
     "build": "tsc",
-    "postinstall": "typings install",
     "install_webdriver": "./node_modules/.bin/webdriver-manager update"
   },
+  "files": [
+    "dist",
+    "LICENSE"
+  ],
   "repository": {
     "type": "git",
     "url": "git+https://github.com/angular/indy.git"

